### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by gohabereg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "postcss-nested-ancestors": "^2.0.0",
+    "raw-loader": "^4.0.2",
+    "style-loader": "^2.0.0",
+    "ts-loader": "^9.1.0",
+    "typescript": "^4.2.4",
+    "webpack": "^5.35.0",
+    "webpack-cli": "^4.6.0"
+  },
+  "dependencies": {
+    "@codexteam/shortcuts": "^1.1.1"
```
Comment (talyguryn) [inline package.json:42]: need to move this to devdeps too, as i understand. because you are going to use bundle not sources

**Review by talyguryn — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gohabereg) [reply package.json:42]: It is included to bundle, so should be in deps

**Comment by gohabereg** [comment]
Tests will not work on CI until core branch is merged to develop and published to npm 😢

File: README.md [hunk 2]
```diff
@@ -1,2 +1,129 @@
...
+```
+
+Include module in your application
+
+```javascript
+const FootnotesTune = require('@editorjs/footnotes');
+```
+
+### Load from CDN
+Load the script from [jsDelivr](https://www.jsdelivr.com/package/npm/@editorjs/footnote) CDN and connect to your page.
```
Comment (neSpecc) [inline README.md]:
```suggestion
Load the script from [jsDelivr](https://www.jsdelivr.com/package/npm/@editorjs/footnotes) CDN and connect to your page.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,129 @@
...
+
+<script src="https://cdn.jsdelivr.net/npm/@editorjs/footnote@latest"></script>
```
Comment (neSpecc) [inline README.md]:
```suggestion
<script src="https://cdn.jsdelivr.net/npm/@editorjs/footnotes@latest"></script>
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,129 @@
...
+`tune.footnote` property contains array of strings respectively to each sup element in Block's content
+
+```json
+{
+  "type": "paragraph",
+  "data": {
+    "text": "You text with a footnote<sup data-tune=\"footnote\">1</sup>. And another footnote<sup data-tune=\"footnote\">2</sup>"
+  },
+  "tunes": {
+    "footnote": [
```
Comment (neSpecc) [inline README.md]: Footnote or Footnotes? Maybe we need to choose one version and rename other places?

File: package.json [hunk 5]
```diff
@@ -3,21 +3,25 @@
...
+    "lint": "eslint src/ --ext .ts",
+    "test": "yarn build && cypress run"
   },
   "devDependencies": {
     "@babel/core": "^7.13.16",
     "@babel/preset-env": "^7.13.15",
+    "@cypress/code-coverage": "^3.9.9",
     "@cypress/webpack-preprocessor": "^5.9.1",
-    "@editorjs/editorjs": "../editor.js-copy",
+    "@editorjs/editorjs": "https://github.com/codex-team/editor.js#feature/tunes-improvements",
```
Comment (neSpecc) [inline package.json]: do not forget to remove branch before merge

**Review by talyguryn — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**